cmake/modules/FindMySQL.cmake

```diff
@@ -26,17 +34,22 @@ if(MYSQL_CONFIG_EXECUTABLE)
   execute_process(COMMAND ${MYSQL_CONFIG_EXECUTABLE} --libs OUTPUT_VARIABLE MYSQL_LIBRARIES OUTPUT_STRIP_TRAILING_WHITESPACE)
 else()
   find_path(MYSQL_INCLUDE_DIR mysql.h
-    /usr/local/mysql/include
-    /usr/local/include/mysql
-    /usr/local/include
-    /usr/include/mysql
-    /usr/include
-    /usr/mysql/include
-    $ENV{MYSQL_DIR}/include
-  )
+    PATH_SUFFIXES include include/mysql include/mariadb
```

Comment:
This only needs to be `PATH_SUFFIXES mysql mariadb`, right? CMake should add `include` by default.

Reply:
It should indeed add `include` by default, but it is not clear to me that any specified `PATH_SUFFIX` be additive to CMake's default (`include` in this case`). So while I think it is reasonable to remove the standalone `include` from the list here, I think the other two entries should remain unchanged.